test/policy_cq_config.hcl

```diff
@@ -0,0 +1,21 @@
+// Configuration AutoGenerated by CloudQuery CLI
+cloudquery {
+  plugin_directory = "./cq/providers"
+  policy_directory = "./cq/policies"
+
+  provider "azure" {
+    source  = "cloudquery/azure"
+    version = "latest"
+  }
+
+  connection {
+    dsn = "host=localhost user=postgres password=pass database=postgres port=5432 sslmode=disable"
+  }
+}
+
+// All Provider Configurations
+provider "azure" {
+  configuration {}
+  resources = [
+  "*"]
+}
```

Comment:
add policy section below

Reply:
We don't actually need a policy section below, since this is the provider repo - we just use 
`cloudquery policy run aws`

See the tests in this PR https://github.com/cloudquery/.github/pull/88